mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -186,14 +197,17 @@ def build_generic(self, *,
         inst : Any
             A new instance of the `cls` class.
         """
+        tunables_path = config.get("include_tunables")
+        if tunables_path is not None:
+            tunables = self._load_tunables(tunables_path, tunables)
```

Comment:
Adds support for referencing `include_tunables` in non-Environments.

Reply:
Useful for the standalone mock optimizer and oneshot optimizer tests for instance.